scholia/app/templates/chemicals_missing.html

```diff
@@ -0,0 +1,309 @@
+SELECT ?wikis ?compound ?compoundLabel ?formula WITH {
+  SELECT (SAMPLE(?wikis_) AS ?wikis) ?compound WHERE {
+    ?compound wdt:P31 wd:Q11173.
+    FILTER(NOT EXISTS {?compound wdt:P2117 []})
+    OPTIONAL { ?compound wikibase:sitelinks ?wikis_ }
+  } GROUP BY ?compound ORDER BY DESC(?wikis)
+  LIMIT 100
+} AS %CHEMICALS WHERE {
+  INCLUDE %CHEMICALS
+  OPTIONAL { ?compound wdt:P274 ?formula }
+  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
+} ORDER BY DESC(?wikis)
+`
+
+missingIDLH = `
+SELECT ?wikis ?compound ?compoundLabel ?formula WITH {
+  SELECT (SAMPLE(?wikis_) AS ?wikis) ?compound WHERE {
+    ?compound wdt:P31 wd:Q11173.
+    FILTER(NOT EXISTS {?compound wdt:P2129 []})
+    OPTIONAL { ?compound wikibase:sitelinks ?wikis_ }
+  } GROUP BY ?compound ORDER BY DESC(?wikis)
+  LIMIT 100
+} AS %CHEMICALS WHERE {
+  INCLUDE %CHEMICALS
+  OPTIONAL { ?compound wdt:P274 ?formula }
+  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
+} ORDER BY DESC(?wikis)
+`
+
+$(document).ready(function() {
+     sparqlToDataTable(missingBoilingPointsSparql, "#missing-boiling-points");
+     sparqlToDataTable(missingMeltingPointsSparql, "#missing-melting-points");
+     sparqlToDataTable(missingMassSparql, "#missing-mass");
+     sparqlToDataTable(missingSolubilitySparql, "#missing-solubility");
+     sparqlToDataTable(missingEnthalpyOfFormationSparql, "#missing-enthalpy-of-formation");
+     sparqlToDataTable(missingVaporPressureSparql, "#missing-vapor-pressure");
+     sparqlToDataTable(missingStandardMolarEntropySparql, "#missing-molar-entropy");
+     sparqlToDataTable(missingDensitySparql, "#missing-density");
+     sparqlToDataTable(missingChemicalStructureSparql, "#missing-chemical-structure");
+     sparqlToDataTable(missingFlashPointSparql, "#missing-flash-point");
+     sparqlToDataTable(missingpKaSparql, "#missing-pKa");
+     sparqlToDataTable(missingCombustionEnthalpy, "#missing-combustion-enthalpy");
+     sparqlToDataTable(missingIDLH, "#missing-idlh");
+ });
+</script>
+ 
+{% endblock %}
+
+{% block page_content %}
+
+<h1>Missing physicalchemical properties for chemicals</h1>
+
+When adding missing physicochemical properties, please take the following into account:
+
+<ul>
+  <li>use units: <a href="https://www.wikidata.org/wiki/Wikidata:Units">Wikidata:Units</a></li>
+  <li>use references: <a href="https://www.wikidata.org/wiki/Help:Sources">Help:Sources</a></li>
+</ul>
+
+The following links prioritize chemical compounds that are listed in most Wikis
+(Wikipedias, WikiCommons, etc). To go to the Wikidata page for each item, click the name
+below and then on the shown Scholia page for that compound the Q-identifier link at the top.
+
+<h2 id="h1">Missing Boiling Points</h2>
+
+The <a href="https://www.wikidata.org/wiki/Property:P2102">boiling point</a> is the temperature above
+which the compound is liquid. Typical units given include Kelvin and degree Celsius. Because it
```

Comment:
Opportunity to link to "temperature" or "Kelvin" and "Celsius".

Reply:
Linked to Kelvin and Celsius (and linked literature to both of them).